feature/554 - added dtos, enums, and base map

junie-stigdon opened the pull request:
## Pull Request

- Added DTOs
  - EmissionsDTO
  - EmissionsParamsDTO
  - ListDTO
  - PaginationDTO
- Added enums
  - Control
  - State
  - UnitFuelType
  - UnitType
- Added BaseMap file

junie-stigdon committed:
added dtos, enums, and base map

jwhitehead77 commented on src/enums/control.enum.ts:
```diff
@@ -0,0 +1,35 @@
+export enum Control {
```
Control is a little to generic and broad so let's name file control-technology.enum.ts and change enum to ControlTechnology

jwhitehead77 commented on src/dto/emissions.dto.ts:
```diff
@@ -0,0 +1,34 @@
+  corRateMeasureFlg: string;
+  heatInput: number;
+  primaryFuelInfo: string;
+  secondaryFuelInfo: ListDTO;
```
Let's change the name of ListDTO to DelimitedListDTO and then create concrete types that extend DelimitedListDTO for the properties secondaryFuelInfo, so2ControlInfo, partControlInfo, noxControlInfo, hgControlInfo

Example:
export class SecondaryFuelInfoDTO extends DelimitedListDTO {
}

secondaryFuelInfo: SecondaryFuelInfoDTO

this gives us the ability to add to these types later if needed and not have to change the datatype

junie-stigdon committed:
updated DTOs, file and variable names changed to hourly-apportioned-emissions

junie-stigdon committed:
fixed typo

junie-stigdon commented:
## Updated Pull Request

- Removed ListDTO
  - Control-Info already delimited in database, returns string now
- Changed file and variables to hourly-apportioned-emissions
  - Route is now api/emissions-mgmt/apportioned/hourly
